### ISSUE TITLE
Handle potential errors in footer parsing

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice#836]
I've implemented [Error Boundaries](https://reactjs.org/docs/error-boundaries.html) in a few places, but not consistently and not that well. Since we do a lot of data processing on the client, we are prone to errors which may bring the app (client) down. One such place is in the [parsing of footer markdown](https://github.com/nextstrain/auspice/blob/master/src/components/framework/footer.js#L164). The output of this should be wrapped in an Error Boundary (or a simple try/catch used as we parse the data.

**Comment by andressrudqvist** [comment]
Hi James, 

I would like to work on this issue. I think that something like this would do the job.

```js
    let cleanDescription;
    try 
    {
      const rawDescription = marked(metadata.description);
      cleanDescription = sanitizer(rawDescription, sanitizerConfig);
    } catch (error) {
      console.error('Error parsing footer description'+error)      
      cleanDescription = (<h1>Something went wrong.</h1>);
    }
```


Regards,

Andres

**Comment by jameshadfield** [comment]
Yes, I think this would be good 👍 

Let's change `<h1>Something went wrong</h1>` to `<p>There was an error parsing the footer description.</p>` or similar.

**Comment by andressrudqvist** [comment]
Hi James,

I modified the message. I made a branch and  [the modified code is](https://github.com/andressrudqvist/auspice/commit/b874dd339e24c93b586c3f7d37544be55d4a17ac).

**Comment by jameshadfield** [comment]
Great! Can you make a PR into this repo so I can test & merge?

**Comment by andressrudqvist** [comment]
Hi James, the PR is already merged.  As part of this issue would you like me to implement Error Boundaries in other key components?

Andres

**Comment by andressrudqvist** [comment]
For reference this is the PR URL:
https://github.com/nextstrain/auspice/pull/1058

**Comment by jameshadfield** [comment]
Closed by #1058
